test/utils/mochaw.js

```diff
@@ -15,13 +15,13 @@ const asyncTestWrap = fn => done => {
   });
 }
 
-const wrapDescribeFunction = ({pageObjects},fn) => function () {
+const wrapDescribeFunction = ({screens},fn) => function () {
```

Comment:
they are still called pageobjects in the file structure, not sure if it makes clear or not :/